pkg/server/public/handlers/record/create.go

```diff
@@ -29,8 +31,17 @@ func (h *Handler) Create() http.HandlerFunc {
 			utils.ErrorResponse(w, err)
 			return
 		}
+
 		input.DatabaseID = form.DatabaseID
 
+		l.Debug("validating record")
+		if validationErrs := validation.ValidateRecord(&input, form); validationErrs.HasAny() {
```

Comment:
Change to !IsEmpty. Check for other instances of HasAny. Fix naming of HasAny since it's misleading. Perhaps ´HasSome ´